script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -9,48 +9,48 @@ Senpai, you're not a member of any clubs. Why are you heading to school this ear
 Senpai's bag makes a clanking sound.
-I thought it was a tool for club activities, but it seems to be a cleaning tool.
+I thought it was a tool for club activities, but it seems to be cleaning equipment.
 <Page6>
 "Oh, do you want to see what's inside? You're so bold to check out your Senpai's belongings, even though you seem so innocent."
 <Page7>
 Senpai leans in closer as if staring into my face.
 ...Well, her face, her smell, and so many other things are so close that I'm not even sure where to look.
 <Page8>
-"No, no, it's not that, I just thought that if you have a foldable broom or a rake that would be unusual..."
+"No, no, it's not that, I just thought that if you have a foldable broom or a rake, that would be unusual..."
 <Page9>
 "Hoh hoh, I see you also want to see my pruning shears.
 All right. In that case, come by the tea ceremony club after school. You're more than welcome."
 <Page10>
-I felt embarrassed when she said that with a soothing smile.
-Not being able to give a witty reply, all I can say is a weak 'yes'.
+I feel embarrassed when she says such things with a soothing smile.
+Not being able to give a witty reply, all I can answer is a weak 'yes'.
 <Page11>
 "It's a promise then. Between you and me, we have a surplus of snacks purchased with club funds. It's not something that stays fresh long, so I need a healthy boy to help out."
 "I-Is that so? Well, if my schedule permits, I'll certainly take you up on that offer.
-Although getting taken in by food is something a cat would do."
+Although, getting taken in by food is something a cat would do."
 <Page12>
-Her smile is so friendly that I couldn't help but joke a little.
+Her smile is so friendly that I can't help but joke a little.
 <Page13>
-The green ribbon wearing Senpai spoke,
+The Senpai with a green ribbon replies, 
 "It's more like a dog, I think.
-I mean, felines are not good. The ones you can rely on to the end are the disciplined and intelligent canines!"
-She looks pleased with herself and shows a determination that I couldn't quite understand.
+I mean, felines are no good. The ones you can rely on to the end are the disciplined and intelligent canines!"
+She looks pleased with herself and shows a determination that I can't quite understand.
 <Page14>
 Because our destination is the same, the conversation somehow continues.
-We talked about last week's mid-term exams.
+We talk about last week's mid-term exams.
 About the gymnastics festival, which was reduced in scale and only lasted one day.
 About the cafeteria, where the menu has never changed.
-About the controversy between the students and the school over the use of mobile devices, we both sighed, so we changed the topic and laughed about our impressions of a movie we had recently seen.
+About the controversy between the students and the school over the use of mobile devices, which makes us both sigh, so we change the topic and laugh about our impressions of a movie we had recently seen.
```

Comment:
IIRC this page was on the edge of scrolling off the bottom, make sure you playtest this one if you haven't already